task_schemas.md

```diff
@@ -312,7 +312,7 @@ Passages capture document structure, such as the title and abstact sections of a
 ```
 
 ## Speech recognition
-- [Schema Template](nusantara/utils/schemas/speech_recognition.py)
+- [Schema Template](nusacrowd/utils/schemas/speech_recognition.py)
```

Comment:
Since `utils/schemas/speech_recognition.py` is not exist, I think you can just remove this line
```suggestion
```